packages/element-library/src/video/util.js

```diff
@@ -34,3 +33,19 @@ export const getBackgroundStyle = () => {
     maxWidth: 'initial',
   };
 };
+
+export const Video = styled.video`
+  position: absolute;
+  max-width: initial;
+  max-height: initial;
+  height: ${({ height }) => `${height}px`};
+  background-image: ${({ poster }) => `${poster}`};
```

Comment:
IIUC this is the main change you've added?

Shouldn't this be like this though?

```suggestion
  background-image: ${({ poster }) => `url("${poster}")`};
```